test_cases/process_approval.js

```diff
@@ -0,0 +1,215 @@
+export default (folderName, Common) => {
+
+    describe('kitchen-sink', () => {
+
+        beforeEach(() => {
+            Common.RunSQL(cy, folderName, ['process_test-kcs-onCreate-process.sql']);
+        });
+
+        // 1. Can see the message for approval in the inbox
+
+        it.only('Can see the message for approval in the inbox', () => {
+
+            // Go to the tab "Process" > "Order"
+
+            cy.get('[data-cy="dd6f7981-cc7b-457c-b231-742ce85004f8"]')
+              .should('exist')
+              .click();
+
+            // cy.get('[data-cy="ABViewGrid_fdc850c5-8cf3-4d8e-8276-568b9907f62d_datatable"]')
```

Comment:
For commented out code. If we don't need it please remove. If it is something we might need in the future, leave a comment above explaining why it's commented out now.